lib/griddler/amazon_s3_ses/adapter.rb

```diff
@@ -97,7 +97,11 @@ def html_part
       end
 
       def force_body_to_utf_8_string(message_body)
-        message_body.to_s.force_encoding(Encoding::UTF_8)
+        # Fallback to an empty string literal to ensure we don't have nil.to_s which returns a frozen string.
+        # A forzen string errors out with the .force_encoding method.
+        body_string = message_body.present? ? message_body.to_s : ""
+
+        body_string.force_encoding(Encoding::UTF_8)
```

Comment:
This looks a little different than before where we had:
```ruby
return "" unless message_body.present?
```

I think this might be better, because this ensures the UTF-8 encoding is always applied.

Reply:
Perfect, I agree.